File: salarioComBonus.java
import java.util.Scanner;

public class salarioComBonus{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
    
        double salario, montanteVendas;
        String nome = sc.nextLine();
        salario = sc.nextDouble();
        montanteVendas = sc.nextDouble();
        

        System.out.println("TOTAL = R$ "+salario+(montanteVendas*(15/100)));

        sc.close(); 
    }
}

File: salarioComBonus.c
// aceito
#include <stdio.h>

int main(){
    char nome[100];
    double salario, montanteVendas;

    fgets(nome, 100, stdin);
    scanf("%lf %lf", &salario, &montanteVendas);

    //printf("%lf", montanteVendas*(15/100));
    //float quinzePorcento = 15/100;
    //printf("%f", quinzePorcento);
    double resFinal = salario + ((montanteVendas*15)/100);
    printf("TOTAL = R$ %.2lf\n", resFinal);
}